scripts/cluster-install/manager_config.yaml

```diff
@@ -0,0 +1,43 @@
+manager:
+  hostname: <manager host name, e.g. manager2>
+  security:
+    ssl_enabled: true
+    admin_password: admin
+
+rabbitmq:
+  username: cloudify
+  password: 'c10udify'
+  ca_path: /home/centos/install_cluster/certs/ca.pem
+  cluster_members:
+    <first rabbitmq instance-name>:
+      default: <first rabbitmq private-ip>
+    <second rabbitmq instance-name>:
+      default: <second rabbitmq private-ip>
+    <third rabbitmq instance-name>:
+      default: <third rabbitmq private-ip>
+
+postgresql_server:
+  ssl_enabled: true
+  ca_path: /home/centos/install_cluster/certs/ca.pem
+
+  postgres_password: 'somesecretvalue123!'
+  cluster:
+    nodes:
+      - <first db private-ip>
+      - <second db private-ip>
+      - <third db private-ip>
+
+postgresql_client:
+  ssl_enabled: true
+  server_password: 'somesecretvalue123!'
```

Comment:
We could also turn on ssl_client_verification here, or make it an option. It'd require ssl_client_verification: true under this section (postgresql_client), and then the following under ssl_inputs:
  postgresql_client_cert_path: /home/centos/install_cluster/certs/<manager instance-name>_cert.pem
  postgresql_client_key_path: /home/centos/install_cluster/certs/<manager instance-name>_key.pem

Reply:
I haven't thought about it. Is there a benefit from adding it, or in case only the R&D uses this script we shouldn't add it?

Reply:
I think even if this is purely for R&D it'd be worth doing as it makes it less likely that we'll break that functionality (again).

Reply:
Got it, I changed it.